README.md

```diff
@@ -10,7 +10,28 @@ Titanoboa achieves feature parity with the vyper compiler while providing an int
 
 Usage and quickstart are [below](#usage-quick-start). For more detailed documentation, please see the [documentation](https://titanoboa.readthedocs.io/en/latest/index.html).
 
-## Installation
+## Setup dev environment
+Clone titanoboa git repo and change into the newly created directory.
```

Comment:
why is this different from the "latest dev version" installation instructions below?

Reply:
Well, I'm not very familiar with python and python packaging. As I understand it, does the section "latest dev version" not explain on how to prepare a dev environment but rather how to get the latest dev version of the project.

Reply:
in that case, this should not replace the "Installation" section but rather be a section further down below like "Setting up a development environment for contributing to titanoboa"

Reply:
Please review the latest commit